### PR TITLE
Update anaconda-project to get rid of errors in census example

### PR DESCRIPTION
**Pull request by ablythed**
Updated the notebook and anaconda-project to get rid of errors. I was unable to get spatialpandas to work, so right now the notebook just uses dd.read_parquet (I deleted the sentence in the description about spatial indexing). This should fix https://github.com/pyviz-topics/examples/issues/160.

### CODE REVIEW

**Comment by jlstevens** [comment]
Looks good to me, at least until spatialpandas is released again.

**Comment by jbednar** [comment]
Can you add `notebook` to the anaconda-project.yml, so that anaconda-project knows that jupyter is available? (It's available anyway through a dependency, but it's not explicit.)  Also, I wasn't able to instantiate the locked environment until I deleted all packages with `cffi` in their names; maybe it needs re-locking?

**Comment by ablythed** [comment]
@jbednar Made those changes and it should work now.

File: census/census.ipynb [hunk 1]
```diff
@@ -1,5 +1,15 @@
 {
  "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Census\n",
+    "Written by James A. Bendar<br>\n",
```
Comment (jbednar) [inline census/census.ipynb:8]:
```suggestion
    "Written by James A. Bednar<br>\n",
```

**Review by jbednar — APPROVED** [review]

**Pull request merged**